spacepy/pybats/rim.py

```diff
@@ -246,16 +229,20 @@ def calc_j(self):
         Calculate total horizontal current as related values.  Each will be
         stored into *self* using the typical key-value approach.  Calculations
         are done for both the northern and southern hemisphere with the
-        appropriate prefixes ('n_' and 's_') applied to each key.
+        appropriate prefixes ('n' and 's') applied to each key.
```

Comment:
Unless you've checked in the code and there's really no underscore, this is probably wrong. If it's trying to render as italic make it code:
```
``n_s``
```

Reply:
The problem with these was that sphinx does not like things ending in an underscore. I tried a few ways to leave the underscore there, but sphinx always gave back warnings. I figured the 'n' is still technically a prefix, which is why my fix was pretty trivial.

Reply:
Yeah, but that doesn't tell the user what the prefix _really is_. Wrapping in double backticks should do the trick--I tested to confirm it in this situation.

Reply:
resolved